src/control/cmd/dmg/pretty/storage_nvme.go

```diff
@@ -227,6 +234,12 @@ func PrintNvmeControllers(controllers storage.NvmeControllers, out io.Writer, op
 		row[fwTitle] = ctrlr.FwRev
 		row[socketTitle] = fmt.Sprint(ctrlr.SocketID)
 		row[capacityTitle] = humanize.Bytes(ctrlr.Capacity())
+		roles := "???"
```

Comment:
Instead of ???, the role data could be used for md-on-scm ?

Reply:
as above